samples/CosmosDBIntegration/CosmosDbIntegrationTutorial.md

```diff
@@ -0,0 +1,117 @@
+# Tutorial: Setup Redis Triggers for Azure Function in Visual Studio
+Article 08/08/2023 Nadia Bugarin, Riley MacDonald, Phoebe Owusu
+
+In this tutorial, you learn how to:
+1. Install packages on an existing Azure function
+2. Use a Redis trigger with your Azure Function
+3. Deploy an Azure Function
+
+
+## Prerequisites
+- Accounts
+    - Azure Redis Cache 
+   - CosmosDB Account
+- Completion of a previous Tutorial
+    - [How to configure Azure Cache for Redis](https://learn.microsoft.com/en-us/azure/azure-cache-for-redis/cache-configure)
+    - [Create an Azure Cosmos DB account, database, container, and items from the Azure portal](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/quickstart-portal)
+        - Finish steps up to (and including), "Add a database and a container."
+    - [Get started with Azure Functions triggers in Azure Cache for Redis](https://learn.microsoft.com/en-us/azure/azure-cache-for-redis/cache-tutorial-functions-getting-started)
+        - Finish steps up to (and including), "Configure the cache."
+
+- Sample code
+    - ReadThrough
+        - [PubSub.Sample](ReadThroughSamples/PubSubSample.cs)
+        - [List.Sample](ReadThroughSamples/ListSample.cs)
+    - WriteAround
+        - [PubSub.Sample](WriteAroundSamples/PubSubSample.cs)
+        - [List.Sample](WriteAroundSamples/ListSample.cs)
+        - [Stream.Sample](WriteAroundSamples/StreamSample.cs)
+    - WriteBehind
+        - [PubSub.Sample](WriteBehindSamples/PubSubSample.cs)
+        - [List.Sample](WriteBehindSamples/ListSample.cs)
+        - [Stream.Sample](WriteBehindSamples/StreamSample.cs)
+    - WriteThrough
+        - [PubSub.Sample](WriteThroughSamples/PubSubSample.cs)
+        - [Stream.Sample](WriteThroughSamples/StreamSample.cs)
+
+## Installing Packages and adjusting variables
```

Comment:
This fits nicely in your first task, Install packages on an existing Azure function. ( I am pretty sure the branded name is Azure Function. (both words uppercase but I would check on it)